packages/preload/src/modules/workspace.ts

```diff
@@ -278,15 +280,14 @@ export async function updateProject(project: Project): Promise<Project> {
       }
     }
 
+    if (updatedProject.updatedAt > project.updatedAt) {
```

Comment:
validation was added to detect whether the method `updateProject` was called by the `sceneUpdate RPC` or manually.